components/Releases/CreateRelease.jsx

```diff
@@ -1,5 +1,12 @@
+import slugify from "slugify"
 import { useState } from "react"
-import { useSupabaseClient } from "@supabase/auth-helpers-react"
+import { useUser, useSupabaseClient } from "@supabase/auth-helpers-react"
```

Comment:
We can grab the `user` from the `useUser` helper so you don't have to pass down the object into components.